homeassistant/components/swiss_public_transport/const.py

```diff
@@ -9,12 +9,19 @@
 CONF_VIA: Final = "via"
 
 DEFAULT_NAME = "Next Destination"
+DEFAULT_UPDATE_TIME = 90
 
 MAX_VIA = 5
 SENSOR_CONNECTIONS_COUNT = 3
+SENSOR_CONNECTIONS_MAX = 15
 
 
 PLACEHOLDERS = {
     "stationboard_url": "http://transport.opendata.ch/examples/stationboard.html",
     "opendata_url": "http://transport.opendata.ch",
 }
+
+ATTR_CONFIG_ENTRY_ID: Final = "config_entry_id"
```

Comment:
I think we should consider adding this to `homeassistant/const.py` to make services more aligned across integrations (in a separate PR though)

Reply:
will make a PR